src/Interfaces/Typeahead/Interface.purs

```diff
@@ -9,9 +9,10 @@ import Control.Promise as Promise
 import Data.Array (head)
```

Comment:
Commenting here arbitrarily. Do any of the changes here affect downstream consumers (I think it's just QuickSight)? If so, we should loop them in on these changes so they're aware of what's happening.